sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Bindings/ConnectionStatesNewtonsoftConverter.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.WebPubSub
+{
+    internal class ConnectionStatesNewtonsoftConverter : JsonConverter<IReadOnlyDictionary<string, BinaryData>>
+    {
```

Comment:
We already have the `BinaryDataJsonConverter`, seems this is not needed.

Reply:
I looked at `BinaryDataJsonConverter`, but it calls `value.ToString()` in `WriteJson`.  I'm not clear on how that'll work with nested objects - would it just stringify nested objects?